resources/windows/wix/properties.wxs.erb

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="windows-1252"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+
+    <ComponentGroup Id="Include_Properties" />
+
+    <!-- Add/Remove Programs icon and help link -->
+    <Icon Id="<%= settings[:company_id] %>.ico" SourceFile="conf\windows\stage\misc\puppetlabs.ico"/>
+    <Property Id="ARPPRODUCTICON" Value="<%= settings[:company_id] %>.ico" />
+    <Property Id="ARPHELPLINK" Value="<%= settings[:HelpLink] %>" />
+    <!-- Save destination folder in Add/Remove programs (ARP) registry key -->
+    <SetProperty Id="ARPINSTALLLOCATION" Value="[INSTALLDIR]" After="CostFinalize" />
+
+    <!-- This property is used in the localization strings for the GUI -->
+    <Property Id="VersionUIString" Value="<%= @version.sub(/\.g[0-9a-z]{7}$/, '') %>" />
+
+    <!--
+      By default, install puppet agent to run as LocalSystem. Password
+      is hidden to prevent it from appearing in msiexec log files
+    -->
+    <Property Id="PUPPET_AGENT_ACCOUNT_DOMAIN" Value="."/>
+    <Property Id="PUPPET_AGENT_ACCOUNT_USER" Value="LocalSystem"/>
+    <Property Id="PUPPET_AGENT_ACCOUNT_PASSWORD" Hidden="yes"/>
+
+    <!--
+      Remembered Property Pattern
+      http://robmensching.com/blog/posts/2010/5/2/The-WiX-toolsets-Remember-Property-pattern
+      -->
+    <!-- Registry settings, remembered install dir is dependent on x86/64.
+         Everything else stays with x86 so both installer types can see it
+      -->
+    <Property Id="INSTALLDIR">
+      <!-- Recall the property in repair, upgrade, and uninstall scenarios -->
+      <RegistrySearch Id="RecallInstallDir" Root="HKLM"
+        Key="SOFTWARE\<%= settings[:company_name] %>\<%= settings[:product_id] %>"
+        Name="<%= settings[:RememberedInstallDirRegKey] %>" Type="raw" Win64="no" />
```

Comment:
Shouldn't `Win64="no"` be `Win64="<%= settings[:win_64]"`, which will either be `yes` or `no` based on what the architecture is? Or should we only conditionally be setting these properties depending on architecture?